src/mbgl/text/glyph_atlas.cpp

```diff
@@ -207,8 +207,8 @@ Rect<uint16_t> GlyphAtlas::addGlyph(GlyphValue& value) {
 }
 
 void GlyphAtlas::removeGlyphValues(GlyphRequestor& requestor, std::map<GlyphID, GlyphValue>& values) {
-    for (auto it = values.begin(); it != values.end(); it++) {
-        GlyphValue& value = it->second;
+    for (auto & it : values) {
```

Comment:
Yeah probably we need to add a few more checks in our `.clang-format` to have that right when applying the changes from `clang-tidy` - I'm gonna manually fix this for now.